src/panels/lovelace/editor/config-elements/hui-conditional-card-editor.ts

```diff
@@ -0,0 +1,274 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  customElement,
+  property,
+  CSSResult,
+  css,
+} from "lit-element";
+import "@polymer/paper-tabs";
+
+import { struct } from "../../common/structs/struct";
+import { HomeAssistant } from "../../../../types";
+import { LovelaceCardEditor } from "../../types";
+import { StackCardConfig } from "../../cards/types";
+import { fireEvent } from "../../../../common/dom/fire_event";
+
+import "../../../../components/entity/ha-entity-picker";
+import "../../../../components/ha-switch";
+
+const conditionStruct = struct({
+  entity: "string",
+  state: "string?",
+  state_not: "string?",
+});
+const cardConfigStruct = struct({
+  type: "string",
+  card: "any",
+  conditions: struct.optional([conditionStruct]),
+});
+
+@customElement("hui-conditional-card-editor")
+export class HuiConditionalCardEditor extends LitElement
+  implements LovelaceCardEditor {
+  @property() public hass?: HomeAssistant;
+  @property() private _config?: StackCardConfig;
+  @property() private _cardTab: boolean = false;
+
+  public setConfig(config: StackCardConfig): void {
+    this._config = cardConfigStruct(config);
+  }
+
+  protected render(): TemplateResult {
+    if (!this.hass || !this._config) {
+      return html``;
+    }
+
+    return html`
+      <paper-tabs
+        .selected=${this._cardTab ? "1" : "0"}
+        @iron-select=${this._selectTab}
+      >
+        <paper-tab
+          >${this.hass!.localize(
+            "ui.panel.lovelace.editor.card.conditional.conditions"
+          )}</paper-tab
+        >
+        <paper-tab
+          >${this.hass!.localize(
+            "ui.panel.lovelace.editor.card.conditional.card"
+          )}</paper-tab
+        >
+      </paper-tabs>
+      ${this._cardTab
+        ? html`
+            <div class="card">
+              ${this._config.card.type
+                ? html`
+                    <div class="card-options">
+                      <mwc-button @click=${this._handleReplaceCard}
+                        >${this.hass!.localize(
+                          "ui.panel.lovelace.editor.card.conditional.change_type"
+                        )}</mwc-button
+                      >
+                    </div>
+                    <hui-card-editor
+                      .hass=${this.hass}
+                      .value="${this._config.card}"
+                      @config-changed="${this._handleCardChanged}"
+                    ></hui-card-editor>
+                  `
+                : html`
+                    <hui-card-picker
+                      .hass="${this.hass}"
```

Comment:
Nit Picky: This is the only `hass` property you have in quotes :)

Reply:
Quotes should be removed :-)

Reply:
Thats what I meant :) sorry for the confusion